iOS/update getting-started guide, removed refs to amplify-app (#2888)
aws-amplify/docs: docs/lib/datastore/fragments/ios/getting-started/30_setupEnv.md

<inline-fragment src="~/lib/datastore/fragments/native_common/setup-env-cli.md"></inline-fragment>

> Once the command is done, a new group named `AmplifyConfig` should be added to your project and contain the following files:

Reviewer: ```suggestion
The `amplify add api` command adds a new `AmplifyConfig` group to your Xcode project. It will contain the following files:
```
